python/agent/crew_ai/src/observability.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from openinference.instrumentation.crewai import CrewAIInstrumentor
+from openinference.instrumentation.langchain import LangChainInstrumentor
+from openinference.instrumentation.litellm import LiteLLMInstrumentor
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk import trace as trace_sdk
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+def setup_observability():
+    headers = {
+        "Galileo-API-Key": os.environ.get("GALILEO_API_KEY", "your_galileo_api_key_here"),
+        "project": os.environ.get("GALILEO_PROJECT", "crew"),  # or "project":"example",
+        "logstream": os.environ.get("GALILEO_LOGSTREAM", "default"),
```

Comment:
```suggestion
        "logstream": os.environ.get("GALILEO_LOG_STREAM", "default"),
```

This matches the Galileo env vars

Reply:
Sure 🤝